lib/segment/src/common/rocksdb_wrapper.rs

```diff
@@ -35,9 +35,10 @@ pub struct LockedDatabaseColumnWrapper<'a> {
     column_name: &'a str,
 }
 
+/// RocksDB options (both global and for column families)
 pub fn db_options() -> Options {
     let mut options: Options = Options::default();
-    options.set_write_buffer_size(DB_CACHE_SIZE);
+    options.set_write_buffer_size(DB_CACHE_SIZE); // write_buffer_size is enforced per column family.
```

Comment:
This buffer size is applied for each column family. It means 10MB per column family. But there is an option `set_db_write_buffer_size` which enables one single value for the whole DB.

If we wanted to set the global buffer size and didn't expect a larger buffer, the option `set_db_write_buffer_size` is preferable. I do not know, did we set this value for expectation of the total buffer size or per-column buffer size.

I propose to keep this option but keep in mind that large amount of column families may use more RAM

Reply:
Right now it uses the default value for `set_db_write_buffer_size` which is 64MB.
So we will shrink memory usage here :)